Dockerfile-build

```diff
@@ -20,3 +20,7 @@ COPY scripts/gen_keys.js scripts/gen_keys.js
 RUN npm install --production && rm -rf ~app/.npm /tmp/*
 
 COPY . /app
+
+USER root
+
+RUN apk del make gcc g++ linux-headers openssl python
```

Comment:
Even though i did this the image stayed at over 100 mb 

![](http://v14d.com/i/5b1a0eb9af634.jpg) mystery

Reply:
Ref: https://github.com/mozilla/fxa-oauth-server/pull/562#discussion_r193951641

Reply:
ah yes because of this https://github.com/gliderlabs/docker-alpine/issues/45